### PR TITLE
GEOT-5134 wfs-ng: better exception reporting

### PR DESCRIPTION
**Pull request by NielsCharlier**

### CODE REVIEW

**Comment by NielsCharlier** [comment]
build passes locally

**Comment by jodygarnett** [comment]
merged locally, rebased so closing by hand.

File: modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/parsers/AbstractGetFeatureResponseParserFactory.java [hunk 1]
```diff
@@ -102,73 +106,114 @@ public boolean canProcess(final WFSRequest request, final String contentType) {
...
             while ((line = reader.readLine()) != null) {
                 head.append(line).append('\n');
             }
-            if(LOGGER.isLoggable(Level.FINER)){
-                System.err.println("Response head:");
-                System.err.println(head);
+        }
+        if (LOGGER.isLoggable(Level.FINER)) {
+            System.err.println("Response head:");
+            System.err.println(head);
```
Comment (bencaradocdavies) [inline modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/parsers/AbstractGetFeatureResponseParserFactory.java]: These ```System.err``` statements flood the logs and cause Travis CI to fail. Something changed in the logic? I am going to replace this block with:
```
if (LOGGER.isLoggable(Level.FINER)) {
    LOGGER.finer("Response head:\n" + head);
}
```

**Comment by bencaradocdavies** [comment]
I pushed my fix to master: "Stop System.err.println flood that breaks Travis CI": https://github.com/geotools/geotools/commit/12844adebde2e5820f7f97f21c2dbd5b95f087f8

**Comment by NielsCharlier** [comment]
Thanks, Ben!

**Pull request closed without merging**